src/checkbox-group.ts

```diff
@@ -211,21 +211,48 @@ export default class GlideCoreCheckboxGroup
 
   override firstUpdated() {
     if (this.disabled) {
-      for (const checkbox of this.#checkboxes) {
+      for (const checkbox of this.#checkboxElements) {
         checkbox.disabled = true;
       }
     }
 
-    this.value = this.#checkboxes
```

Comment:
Setting `value` initially is now the responsibility of [`#onDefaultSlotChanged()`](https://github.com/CrowdStrike/glide-core/pull/765/files#diff-a2c28e4def2289fe312c1ebced684905678bdf6ff0498983ceb13422cc50b047R571-R577).